sass/themes/donate/donate-cr/donate-cr.scss

```diff
@@ -31,10 +31,10 @@
 @import "../../../base/components/header/base-header";
 @import "../../../base/components/navigation/footer-nav";
 @import "../../../base/components/footer/footer";
+@import "../../../base/components/form/standard-form";
```

Comment:
Do you mean to update the donate form styling? It still gets overridden by the default payin form styling below, but if we do want to update the donate form styling we'll need to test that thoroughly in donate.